helmfile/charts/prometheus-alerts/templates/rules/predict-linear.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.predictLinear}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "predict-linear" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: predict-linear
+    rules:
+    - alert: FileVolumeEightyProcentInThreeDays
+      annotations:
+        message: Filesystem usage is predicted to go over eighty percent within the next 3 days at current write rate.
+      expr: predict_linear(node_filesystem_avail_bytes{job="node-exporter",fstype!=""}[24h], 3*24*3600) <= (node_filesystem_size_bytes*0.20)
+      for: 5m
+      labels:
+        severity: warning
+    - alert: PersistentVolumeEightyProcentInThreeDays
+      annotations:
+        message: Persistentvolume usage is predicted to go over eighty percent within the next 3 days at current write rate.
+      expr: predict_linear(kubelet_volume_stats_available_bytes{job="kubelet"}[24h], 3*24*3600) <= (kubelet_volume_stats_capacity_bytes*0.20)
+      for: 5m
+      labels:
+        severity: warning
+    - alert: MemoryEightyProcentInThreeDays
+      annotations:
+        message: Memory usage is predicted to go over eighty percent within the next 3 days at current use rate.
+      expr: predict_linear(node_memory_MemAvailable_bytes[24h], 3*24*3600) <= (node_memory_MemTotal_bytes*0.20)
```

Comment:
All these will be per node if I understand it correctly? Could you then add the node name in question in the message as well?
_This applies for all the alerts_